### DISCUSSION TITLE
9.0.0

### DISCUSSION THREAD

**Discussion by DuCanhGH** [serwist/serwist#66] [Announcements]
See https://serwist.pages.dev/blog/2024/03/10/serwist-v9.

Serwist 9.0.0 will be released on 2024/04/21.

Two months in the making.

(definitely not me slacking off)

**Comment by FleetAdmiralJakob** [comment]
Created a Next.js issue for recommending an ESM Config file: https://github.com/vercel/next.js/issues/61720

**Comment by dhamibirendra** [comment]
Hi @DuCanhGH 
Great work! any news when this version will be released? 😃

**Reply by DuCanhGH** [reply]
@dhamibirendra I intended to release it last week, but I've found myself a bit too busy to work on the last few things. Sorry about that... You can use Serwist preview right now though :)

A potential release window would be this weekend.

**Reply by dhamibirendra** [reply]
Awesome! @DuCanhGH. Thanks!

**Reply by DuCanhGH** [reply]
@dhamibirendra I believe that the release will need to be held back for a while. At the moment, there are a few features that I want to implement for 9.0.0, such as built-in support for subresource integrity metadata. Hopefully we can get everything done right this week.

**Reply by rortan134** [reply]
Please take your time. You've got an awesome library, thank you for your work.

**Reply by dhamibirendra** [reply]
Thanks @DuCanhGH 
No worries! Keep up the awesome work :)

**Comment by DuCanhGH** [comment]
@dhamibirendra @rortan134 Serwist v9 has been released. Thank you for having waited!

**Reply by rortan134** [reply]
Amazing! thank you.

**Reply by dhamibirendra** [reply]
Awesome!!! Thanks @DuCanhGH 👍

**Reply by FleetAdmiralJakob** [reply]
You are the best. Thank you for all your awesome work you put into this project and next-pwa
